lld/test/COFF/lto-imp-prefix.ll

```diff
@@ -37,3 +39,19 @@ define void @other() {
 entry:
   ret void
 }
+
+;--- other2.ll
+target datalayout = "e-m:w-p270:32:32-p271:32:32-p272:64:64-i64:64-i128:128-f80:128-n8:16:32:64-S128"
+target triple = "x86_64-w64-windows-gnu"
+
+@__imp_importedFunc = global ptr @importedFunc
+
+define void @importedFunc() {
```

Comment:
Perhaps add a comment to emphasize that unprefixed symbol is external.

Reply:
Sure, to clarify that this testcase explicitly tests providing two external symbols from the same object file, with/without prefix.